test/services/audit/AuditServiceSpec.scala

```diff
@@ -46,15 +46,15 @@ class AuditServiceSpec extends AuditTestSupport with OptionValues {
     }
 
     "audit full payload" in {
-      auditService.auditAllPagesUserInput(cacheMapAllRecords)
+      auditService.auditAllPagesUserInput(Json.toJson(cacheMapAllRecords.data).as[JsObject])
```

Comment:
Similar here, can you use `Json.obj(cacheMapAllRecords.data)`.

The same comment in line 57

Reply:
its taking a Map and then converting into varargs tuple